### DISCUSSION TITLE
Allow cloning over https

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#836] [Feature Requests]
We currently do not allow cloning over https. We'll need some secure way of storing the username and password in order to provide this. Ideally the user shouldn't ever be providing us with a password, but rather a deploy token of some form.

We should make it clear that this will not work with 2 factor authentication.

```
Personal Notes -
* https://mail.google.com/mail/u/2/#inbox/FMfcgzGkXwLJcrCGQbfzWDMnQQnZFFdG
```

**Comment by LecrisUT** [comment]
What is the current priority and timeline for implementing this?

Most git repositories support authenticating using token or oauth, and gitjournal already stores the ssh credentials so it should be equally or in the case of selfhosted gitea, more secure to implement either of those 2.

**Comment by andrewdunndev** [comment]
If there was a way to incentivize this financially I would happily jump in. This is preventing me from using gitjournal (I've purchased the app to attempt to be supportive).
